docs/source/_src_user_guide/state_propagation/propagation_setup/dynamics_types/multi_type.rst

```diff
@@ -7,7 +7,7 @@ Multi-type dynamics
 Tudat permits the propagation of any combination of types of dynamics, for any number of bodies
 One example is the simulation of coupled translational-rotational dynamics of one or more bodies, or the combined translational and mass dynamics of a body (e.g. spacecraft under thrust).
 
-To define multi-type propagator settings, you must first define the propagator settings for each type of dynamics separately, after which you combine these using the function as follows: 
+To define multi-type propagator settings, you must first define the propagator settings for each type of dynamics separately, after which you combine these using the function as follows using the :func:`~tudatpy.numerical_simulation.propagation_setup.propagator.multitype` function: 
 
     .. tabs::
 
```

Comment:
'Vehicle' isn't mentioned in the code example. This could lead to unnecessary confusion, so maybe omitting it or including it in the code example would be better.

Reply:
You' ll see the vehicle (I think) if you expand the full code (which includes the whole setup of the propagator). Perhaps this should be expanded by default, but then it would get very long. I may indeed be more clear to remove this mention of vehicle, but then the priblem is still tha the code example (without the expanded stuff) is not very clear... Perhaps move more into the example, and remove the expanded option?